core/src/main/scalajvm/sttp/tapir/static/TapirStaticContentEndpoints.scala

```diff
@@ -36,8 +36,10 @@ trait TapirStaticContentEndpoints {
   private val ifModifiedSinceHeader: EndpointIO[Option[Instant]] = optionalHttpDateHeader(HeaderNames.IfModifiedSince)
   private val lastModifiedHeader: EndpointIO[Option[Instant]] = optionalHttpDateHeader(HeaderNames.LastModified)
   private val contentTypeHeader: EndpointIO[Option[MediaType]] = header[Option[MediaType]](HeaderNames.ContentType)
+  private val contentLengthHeader: EndpointIO[Option[Long]] = header[Option[Long]](HeaderNames.ContentLength)
```

Comment:
The binary compatibility check failed because of the introduction of `contentLengthHeader` and `acceptRangesHeader`. I might be missing something, but I think no harm is done here.

Reply:
we cannot add `val`s to traits; we can add `def`s though, if needed

Reply:
Do I understand correctly, that not only the ones I introduced but all `val`s defined in this trait are invalid and should be defined as `def`s (if needed)?

Reply:
No, the ones that are already `val`s have to stay this way (due to bin compat). If we want to add new definitions in a binary-compatible way, we need to add `def`s